### PR TITLE
new package: gdrive-downloader

### PR DESCRIPTION
**Pull request by Freed-Wu**

### CODE REVIEW

File: packages/gdrive-downloader/build.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/Akianonymus/gdrive-downloader
+TERMUX_PKG_DESCRIPTION="Download a gdrive folder or file easily, shell ftw."
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=2022.03.31
+TERMUX_PKG_SRCURL=$TERMUX_PKG_HOMEPAGE.git
+TERMUX_PKG_GIT_BRANCH=master
```
Comment (xtkoba) [inline packages/gdrive-downloader/build.sh:7]: You should specify a commit hash. Without one the source tree may change for every fetch.

**Review by xtkoba — CHANGES_REQUESTED** [review]

File: packages/gdrive-downloader/build.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/Akianonymus/gdrive-downloader
+TERMUX_PKG_DESCRIPTION="Download a gdrive folder or file easily, shell ftw."
+TERMUX_PKG_LICENSE="GPL-3.0"
```
Comment (xtkoba) [inline packages/gdrive-downloader/build.sh:3]: The license for this software should be the "Unlicense" license.

**Review by xtkoba — CHANGES_REQUESTED** [review]

**Comment by Freed-Wu** [comment]
Thanks. `$_COMMIT` has been added.

**Comment by Freed-Wu** [comment]
Fix typo of license.

**Review by xtkoba — CHANGES_REQUESTED** [review]
Please have this depend also on `curl`.

**Comment by Freed-Wu** [comment]
Add curl

**Review by xtkoba — APPROVED** [review]

**Pull request merged**